core/base/src/TStyle.cxx

```diff
@@ -400,13 +400,19 @@ TStyle::~TStyle()
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-/// Copy constructor.
+/// Copy constructor and assignment operator.
 
 TStyle::TStyle(const TStyle &style) : TNamed(style), TAttLine(style), TAttFill(style), TAttMarker(style), TAttText(style)
 {
    ((TStyle&)style).Copy(*this);
 }
 
+TStyle& TStyle::operator=(const TStyle& style)
+{
+   ((TStyle&)style).Copy(*this);
+   return *this;
```

Comment:
Seems to be, casting `((TStyle&)style)` not necessary here.
One can just call `style.Copy(*this);`. Same in copy constructor.
Probably, casting remain from the past when Copy method had no `const` specifier.

Reply:
Thanks for the review! :) Yes, you are right, I've put on top a second commit optimizing the code.